mne/surface.py

```diff
@@ -1426,6 +1426,8 @@ def _find_nearest_tri_pts(rrs, pt_triss, pt_lens,
         pqs = np.empty((len(drs), 2))
         dists = np.empty(len(drs))
         dist = np.inf
+        # make life easier for numba var typing
+        p, q, pt = np.float64(0.), np.float64(1.), np.int64(0)
```

Comment:
Figured out how to actually fix it. Also opened upstream bug:

https://github.com/numba/numba/issues/5584

Will merge once CIs come back happy